src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+            List<Hotel> hotelList = hotelController.findHotelByName(hotelName);
+            if (hotelList.isEmpty()) {
+                System.out.println("Hotel " + hotelName + " not found.");
+            } else {
+                System.out.println("The list of found hotels by name: " + hotelName);
+                VisualUserMenu.printListInConsole(null, hotelList);
+            }
+            return hotelList;
+        }
+    }
+
+    private List<Hotel> findHotelByCityMenu() {
+        while (true) {
+            String city = visualUserMenu.getValidInputFromUser("Enter city (enter \"0\" to return back to menu): ",
+                InputType
+                    .STRING);
+            if (city.equals("0")) {
+                return null;
+            }
+            VisualUserMenu.outputSplitLine();
+            List<Hotel> hotelList = hotelController.findHotelByCity(city);
+            if (hotelList.isEmpty()) {
+                System.out.println("Hotel in " + city + " not found.");
+            } else {
+                System.out.println("The list of found hotels in: " + city);
+                VisualUserMenu.printListInConsole(null, hotelList);
+            }
+            return hotelList;
+        }
+    }
+
+    private Map<String, List<Room>> findRoomMenu() {
+        String city = visualUserMenu.getValidInputFromUser("Enter city: ", InputType.STRING);
+        Integer persons = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of persons: ",
+            InputType.INTEGER));
+        Integer price = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter maximum price: ", InputType
+            .INTEGER));
+        String reservedFrom;
+        String reservedTo;
+        Date arrival;
+        Date departure;
+        for ( ; ; ) {
+            reservedFrom = visualUserMenu.getValidInputFromUser("Enter arrival date in the format \"dd.mm.yyyy\": ",
+
+                InputType.DATE);
+            reservedTo = visualUserMenu.getValidInputFromUser("Enter the departure date in the format \"dd.mm"
+                + ".yyyy\": "
+                + "", InputType.DATE);
+            DateFormat df = new SimpleDateFormat("dd.MM.yyyy");
+            arrival = new Date(0, 0, 0);
+            departure = new Date(0, 0, 1);
+            try {
+                arrival = df.parse(reservedFrom);
+                departure = df.parse(reservedTo);
+            } catch (ParseException e) {
+                log.error("Error parsing input date to Date format", e);
+            }
+            if (arrival.after(departure) || arrival.before(new Date())) {
+                System.out.println("Arrival date must exceed current date but must come before departure!");
+                continue;
+            }
+            break;
+        }
+        Map<String, List<Room>> foundRooms = hotelController.getAvailableRooms(city, persons, price, arrival,
+            departure);
+        System.out.println("The list of rooms found:\n ");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)